app/screens/entry/entry.js

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+/* eslint-disable global-require*/
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {
+    View,
+    AppState,
+    Platform,
+} from 'react-native';
+
+import DeviceInfo from 'react-native-device-info';
+
+import {Client4} from 'mattermost-redux/client';
+import EventEmitter from 'mattermost-redux/utils/event_emitter';
+
+import {
+    app,
+    store,
+} from 'app/mattermost';
+import {ViewTypes} from 'app/constants';
+import PushNotifications from 'app/push_notifications';
+import {stripTrailingSlashes} from 'app/utils/url';
+
+import EmptyToolbar from 'app/components/start/empty_toolbar';
+import Loading from 'app/components/loading';
+import SafeAreaView from 'app/components/safe_area_view';
+import StatusBar from 'app/components/status_bar';
+
+const lazyLoadSelectServer = () => {
+    return require('app/screens/select_server').default;
+};
+
+const lazyLoadChannel = () => {
+    return require('app/screens/channel').default;
```

Comment:
I think this will affect unbundling cause it will not whitelist this files from the main bundle, anyway I can take care of that